newdle/client/src/components/Footer.module.scss

```diff
@@ -0,0 +1,60 @@
+@import '../styles/palette.scss';
+
+.footer {
+  min-width: 100vw;
+  bottom: 0;
+  background: linear-gradient($pink, $coral);
+  padding: 1.5em 3em 1.5em 3em;
+  vertical-align: middle;
+
+  .info {
+    color: $white;
+
+    a {
+      color: $white;
+      font-weight: bold;
+    }
+
+    .newdle {
+      margin-bottom: 0;
+    }
+
+    .cern {
+      opacity: 0.8;
+    }
+  }
+
+  .links {
+    text-align: center;
+    padding: 0;
+
+    li {
+      display: inline;
+
+      .link {
+        font-size: 14px;
+        color: $white;
+        opacity: 1;
+      }
+
+      + li ::before {
```

Comment:
Shouldn't this be `& + li::before`?

`::before` is not a child element of the `li` but  but actually part of it

Reply:
Right!
About the ampersand, AFAIK it can be omitted, is there any reason to add it? readability maybe?

Reply:
you are right, there's no need for `&` here, since the rule is `foo + bar` and not `foo+bar`